src/main/java/usersandpostsCRUD/demo/entity/City.java

```diff
@@ -0,0 +1,49 @@
+package usersandpostsCRUD.demo.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+public class City {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+    private String postCode;
```

Comment:
Post code in macedonia is Long. Try to convert the string that you accept in the request body, to convert it to long. If it cannot be converted, throw a meaningful exception message